### DISCUSSION TITLE
Issue Regarding Joining the GitHub Campus Experts Program

### DISCUSSION THREAD

**Discussion by preetshah007** [community/community#128139] [GitHub Education]
### Hi everyone,

For GitHub's Campus Expert Program on Visiting the website and clicking the link (https://education.github.com/campus_experts/join) after Logging in it showed that the "Page isn't Working". I have attached the Screenshot.
![SS](https://github.com/community/community/assets/136613427/e10150bd-af3c-44c6-a4e3-bb036e477641)

Could you please check and let me know?

**Comment by buu-huu** [comment]
The link is working for me. Try to delete browser cookies, website data, etc. or try it in incognito mode. That should fix the issue.

**Reply by preetshah007** [reply]
On trying it on Incognito: 
![Screenshot 2024-06-11 195720](https://github.com/community/community/assets/136613427/f89dcbca-9bc0-4b65-969f-9250f5648eba)

**Reply by buu-huu** [reply]
Can you try these links

https://education.github.com/campus_experts
https://education.github.com/learner/learn

But campus experts are currently closed, as far as I can see:

![grafik](https://github.com/community/community/assets/55555828/eb84e102-ddf6-456d-af13-a9533f2051b3)

If you're interested in the student pack only, it should work. At least, for me it's working:

![grafik](https://github.com/community/community/assets/55555828/d13a0e46-6b0a-4a1c-85ee-62f0d2f00582)

**Reply by preetshah007** [reply]
Okay got it thank you very much!

**Comment by samus-aran** [comment]
Hi @preetshah007 👋 

Are you currently logged in and having these issues?

**Reply by preetshah007** [reply]
It is now sorted. Thank You very much!

**Reply by samus-aran** [reply]
Awesome, Glad to hear it! 🎉 Thank you for returning and providing an answer 👍
